charts/opensearch-operator/templates/opensearch-operator-controller-manager-deployment.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
+        - --watch-namespace={{ .Values.manager.watchNamespace }}
```

Comment:
If we don't want to show this env setting if `watchNamespace` is not set, we can add the if statement around this line.
```
{{- if . Values.manager.watchNamespace }}
        - --watch-namespace={{ .Values.manager.watchNamespace }}
{{- end }}
```